pkg/helm/chartproxy/repos.go

```diff
@@ -192,7 +201,9 @@ func (b *helmRepoGetter) List() ([]*helmRepo, error) {
 			klog.Errorf("Error unmarshalling repo %v: %v", item, err)
 			continue
 		}
-		helmRepos = append(helmRepos, helmConfig)
+		if helmConfig != nil {
```

Comment:
see above, then this becomes
if !helmConfig.disabled {
.
.

}
```

```

Reply:
Done